src/services/loopd/loop-service.ts

```diff
@@ -41,6 +43,9 @@ export const LoopService = ({
   lndInstanceName,
 }: LoopdConfig): ISwapService => {
   const mac = Buffer.from(macaroon, "base64").toString("hex") as Macaroon
+  if (!tlsCert) {
```

Comment:
probably here should be the validation, also service should not throw exceptions. Please check onchain service as example